src/js/components/ShareButtons.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Please update to the latest version of React before starting to use hooks

Reply:
yea was waiting for the thing to be merged